components/dashboard/src/components/RepositoryFinder.tsx

```diff
@@ -157,3 +189,25 @@ function saveSearchData(searchData: string[]): void {
         console.warn("Could not save search data into local storage", error);
     }
 }
+
+function filterRepos(searchString: string, suggestedRepos: SuggestedRepository[]) {
+    let results = suggestedRepos;
+    const normalizedSearchString = searchString.trim().toLowerCase();
+
+    if (normalizedSearchString.length > 1) {
+        results = suggestedRepos.filter((r) => {
+            return `${r.url}${r.projectName || ""}`.toLowerCase().includes(normalizedSearchString);
```

Comment:
**issue**: Not sure if this is the best line to comment, but I just imported a repository as project and is listed at the bottom of the list. Shouldn't we always list projects at the top?

<img width="517" alt="Screenshot 2023-09-14 at 20 04 13" src="https://github.com/gitpod-io/gitpod/assets/120486/a3303021-04c0-4c30-b08f-6d3c6d46df7a">

Reply:
Yah, I've updated the logic to move projects above personal repos now (even if you haven't started the project recently). This differs from how the logic is in prod today, but seems like everyone agrees they should be above personal repos, so going with that now.